06_neighborhoods/Transportation/transit_trips_cost_city.R

```diff
@@ -0,0 +1,292 @@
+# collapse to places and also create data quality marker
+# data quality can be 1 when most of the tracts that fall in the place (e.g., >50% of the tracts) 
+# have most of their area falling in the place (e.g., >50% of the tract's area is in the place)
+# otherwise, data quality is 2
+transit_cost_city_2015 <- transit_cost_city_2015 %>% 
+  dplyr::group_by(state, place) %>% 
+  dplyr::summarize(index_transportation_cost = round(mean(t_80ami), 2),
+                   n = n(),
+                   dq = sum(afact > 0.5, na.rm = TRUE)
+  )%>%
+  dplyr::mutate(index_transportation_cost_quality = ifelse(dq / n > 0.5, 1, ifelse(dq / n <= 0.5, 2, NA)))
+
+
+# left join with places file to get rid of irrelevant places data
+transit_cost_city_2015 <- left_join(places, transit_cost_city_2015, by=c("state","place"))
+# 29317 obs to 486 obs
+
+# keep only the variables we will need & replace NA qual vars where there is NA metric value
+transit_cost_city_2015 <- transit_cost_city_2015 %>% 
+  mutate(year=2015)%>%
+  select(year, state, place, index_transportation_cost, index_transportation_cost_quality)%>%
+  dplyr::mutate(index_transportation_cost_quality = ifelse(is.na(index_transportation_cost), NA, index_transportation_cost_quality))
+
+
+
+# TRANSPORTATION COST 2019
+transit_cost_city_2019 <- left_join(transit_cost_tracts_2019, tract_place, by=c("state", "county","tract"))
+# collapse to places and also create data quality marker
+transit_cost_city_2019 <- transit_cost_city_2019 %>% 
+  dplyr::group_by(state, place) %>% 
+  dplyr::summarize(index_transportation_cost = round(mean(t_80ami), 2),
+                   n = n(),
+                   dq = sum(afact > 0.5, na.rm = TRUE)
+  )%>%
+  dplyr::mutate(index_transportation_cost_quality = ifelse(dq / n > 0.5, 1, ifelse(dq / n <= 0.5, 2, NA)))
+
+# left join with places file to get rid of irrelevant places data
+transit_cost_city_2019 <- left_join(places, transit_cost_city_2019, by=c("state","place"))
+# 29317 obs to 486 obs
+# keep only the variables we will need & replace NA qual vars where there is NA metric value
+transit_cost_city_2019 <- transit_cost_city_2019 %>% 
+  mutate(year=2019)%>%
+  select(year, state, place, index_transportation_cost, index_transportation_cost_quality)%>%
+  dplyr::mutate(index_transportation_cost_quality = ifelse(is.na(index_transportation_cost), NA, index_transportation_cost_quality))
+
+
+
+# TRANSIT TRIPS 2015
+transit_trips_city_2015 <- left_join(transit_trips_tracts_2015, tract_place, by=c("state", "county","tract"))
+# collapse to places and also create data quality marker
+# data quality can be 1 when most of the tracts that fall in the place (e.g., >50% of the tracts) 
+# have most of their area falling in the place (e.g., >50% of the tract's area is in the place)
+# otherwise, data quality is 2
+transit_trips_city_2015 <- transit_trips_city_2015 %>% 
+  dplyr::group_by(state, place) %>% 
+  dplyr::summarize(index_transit_trips = sum(transit_trips_80ami),
```

Comment:
Should this be a weighted sum by `afact`? Same comment applies to 2019 trips.

Reply:
Yes, thank you! added